src/si/length.rs

```diff
@@ -39,6 +39,8 @@ quantity! {
         @yoctometer: prefix!(yocto); "ym", "yoctometer", "yoctometers";
 
         @angstrom: 1.0_E-10; "Å", "ångström", "ångströms";
+        // Atomic unit of length (Bohr radius)
+        @atomic_unit_of_length: 5.291_772_109_03_E-11; "a.u. of length", "atomic unit of length", "atomic units of length";
```

Comment:
Is Bohr radius actually used as a unit?

```suggestion
        @bohr_radius: 5.291_772_109_03_E-11; "a₀", "bohr radius", "bohr radiuses";
        /// Atomic unit of length (Bohr radius).
        @atomic_unit_of_length: 5.291_772_109_03_E-11; "a.u. of length", "atomic unit of length",
            "atomic units of length";
```